ush/templates/FV3LAM_wflow.xml

```diff
@@ -834,8 +834,10 @@ MODULES_RUN_TASK_FP script.
     <envar><name>ENSMEM_INDX</name><value><cyclestr>#{{ ensmem_indx_name }}#</cyclestr></value></envar>
   
     <dependency>
-      {%- if do_dacycle and do_nonvar_cldanal%}
+      {%- if do_dacycle and do_refl2tten%}
       <taskdep task="&RADAR_REFL2TTEN_TN;"/>
+      {%- elif do_dacycle and do_nonvar_cldanal%}
+      <taskdep task="&CLDANL_NONVAR_TN;"/>
```

Comment:
If both do_refl2tten and do_nonvar_cldanal are ture, what will happen?

Reply:
I've been thinking on this. To my understanding, our current workflow is GSI->cldvar-> ref2tten -> fcst,  so ref2tten is the last piece before fcts, right?  If no ref2tten, fcst will run after cldvar;  if no cldvar, then fcst will run after GSI; if no GSI, fcst will run after the IC/BC step. Let me know if there is other scenarios I am not aware of. Thanks!